examples/energy-management-app/energy-management-common/common/include/EnergyManagementAppCmdLineOptions.h

```diff
@@ -25,7 +25,10 @@ namespace chip {
 namespace app {
 namespace Clusters {
 namespace DeviceEnergyManagement {
+constexpr EndpointId kEvseEndpoint        = 1;
+constexpr EndpointId kWaterHeaterEndpoint = 2;
 
+EndpointId GetMainAppEndpointId();
```

Comment:
This likely don't belong in a file called `examples/energy-management-app/energy-management-common/common/include/EnergyManagementAppCmdLineOptions.h` when they don't relate to command line options. Suggest moving these elsewhere.

Reply:
Moved to `CommonMain.h` thanks